deploy/kubernetes/helm/che/charts/che-keycloak/templates/ingress.yaml

```diff
@@ -24,7 +24,7 @@ spec:
   tls:
   - hosts:
     - {{ template "keycloakHost" . }}
-    secretName: che-tls
+    secretName: {{ .Values.global.tls.secretName }}
```

Comment:
Good catch.

Looks like one more hardcoded usage of  `che-tls` is missing: [certificate.yaml](https://github.com/i300543/che/blob/master/deploy/kubernetes/helm/che/templates/certificate.yaml#L14)

Reply:
@i300543 Should `cerificate.yaml` be updated with `secretName` from configuration instead of hardcoded one?

Reply:
well, yes it probably should :)
the certificate.yaml only relevant to letsencrypt tls flow, and we dont use it anymore
so it never came up.
i guess i will upload 1 more commit.
this shouldn't require the whole selenium checkup again does it ?

Reply:
I think no, it doesn't require the whole selenium checkup again